server_code/api.py

```diff
@@ -0,0 +1,53 @@
+# SPDX-License-Identifier: MIT
+#
+# Copyright (c) 2024 The Anvil Extras project team members listed at
+# https://github.com/anvilistas/media-manager/graphs/contributors
+#
+# This software is published at https://github.com/anvilistas/media-manager
+import anvil.server
+from anvil.tables import app_tables
+
+from . import helpers
+
+
+def _fetch_media(request, hash, **kwargs):
+    row = app_tables.media.get(hash=hash, verified=True)
+    return row["content"] if row else anvil.server.HttpResponse(404)
+
+
+def _store_media(request, hash=None, verify=None, **kwargs):
+    content = request.body
+    if not content:
+        return
+
+    verify = helpers.boolean_from_string(verify, default=True)
```

Comment:
does this behave differently now?
I think before it True if verify was `None`
now I think that'll return `False`

Reply:
Oops. I forgot to include None!